frontend/src/wizards/Governance/Policy/stable/CM-Configuration-Management/operator-policy-general-install.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: policy.open-cluster-management.io/v1
+kind: Policy
+metadata:
+  name: operator-policy-install-name
+  namespace: open-cluster-management-global-set
+spec:
+  disabled: false
+  policy-templates:
+    - objectDefinition:
+        apiVersion: policy.open-cluster-management.io/v1beta1
+        kind: OperatorPolicy
+        metadata:
+          name: install-operator
+        spec:
+          remediationAction: enforce
```

Comment:
I think we may want to have this default to inform for safety. I think we do this with the other out of the box policy templates.

Reply:
But this title is "Install an operator". That mean we should change other things too

Reply:
@yiraeChristineKim what do you mean by changing other things?

Reply:
We cannot use "Install" words because it doesn't install at all. Instead, it is just "supervise"